libs/src/services/creatorNode/CreatorNodeSelection.ts

```diff
@@ -29,19 +95,23 @@ class CreatorNodeSelection extends ServiceSelection {
     equivalencyDelta = null,
     preferHigherPatchForPrimary = true,
     preferHigherPatchForSecondaries = true
-  }) {
+  }: CreatorNodeSelectionConfig) {
     super({
       getServices: async () => {
-        this.currentVersion = await ethContracts.getCurrentVersion(CREATOR_NODE_SERVICE_NAME)
-        const services = await this.ethContracts.getServiceProviderList(CREATOR_NODE_SERVICE_NAME)
+        this.currentVersion = await ethContracts.getCurrentVersion(
+          CREATOR_NODE_SERVICE_NAME
+        )
+        const services = await this.ethContracts.getServiceProviderList(
+          CREATOR_NODE_SERVICE_NAME
+        )
         return services.map((e) => {
           setSpIDForEndpoint(e.endpoint, e.spID)
           return e.endpoint
         })
       },
       // Use the content node's configured whitelist if not provided
-      whitelist: whitelist || (creatorNode && creatorNode.passList),
-      blacklist: blacklist || (creatorNode && creatorNode.blockList)
+      whitelist: whitelist ?? creatorNode?.passList,
```

Comment:
yeah from what i can tell yes: `whitelist` is either a set, null, or undefined, so ?? knocks off null&undefined, which || would also only do since set is has no falsy value (like 0 or '')